scss/_base_links.scss

```diff
@@ -5,9 +5,18 @@
     text-decoration: none;
     border-bottom: 1px solid;
     font-weight: bold;
+    transition: all .5s;
```

Comment:
`.5` is a long time to wait for a none movement animation. I would suggest no more the `.3s` for this type of animation.